File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        res = []
        exist = [False for _ in range(len(nums))]
        for num in nums:
            if exist[num]:
                res.append(num)
            else:
                exist[num] = True
        return res

File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        vector<int> res;
        vector<bool> exist(nums.size());
        for(int i = 0; i < nums.size(); i++)
        {
            if(exist[nums[i]])
            {
                res.push_back(nums[i]);
            }
            else
            {
                exist[nums[i]] = true;
            }
        }
        return res;
    }
};